compiler/src/dotty/tools/dotc/ast/DesugarEnums.scala

```diff
@@ -2,20 +2,26 @@ package dotty.tools
 package dotc
 package ast
 
-import core._
-import util.Positions._, Types._, Contexts._, Constants._, Names._, NameOps._, Flags._
-import SymDenotations._, Symbols._, StdNames._, Annotations._, Trees._
-import Decorators._
-import collection.mutable.ListBuffer
-import util.Property
-import typer.ErrorReporting._
+import dotty.tools.dotc.ast.Trees._
+import dotty.tools.dotc.core.Constants._
+import dotty.tools.dotc.core.Contexts._
+import dotty.tools.dotc.core.Decorators._
+import dotty.tools.dotc.core.Flags._
+import dotty.tools.dotc.core.Names._
+import dotty.tools.dotc.core.StdNames._
+import dotty.tools.dotc.core.Symbols._
+import dotty.tools.dotc.core.Types._
+import dotty.tools.dotc.core._
+import dotty.tools.dotc.reporting.diagnostic.messages.EnumCannotBeLazy
+import dotty.tools.dotc.typer.ErrorReporting._
+import dotty.tools.dotc.util.Positions._
+import dotty.tools.dotc.util.Property
```

Comment:
FWIW, I usually try to preserve the existing import style and not let IntelliJ do its thing. Not sure if anybody feels strongly on this — tho the new imports seem more boilerplatey.